### ISSUE TITLE
Rename claim() to requestOffer()

### ISSUE THREAD

**Issue by tempe-techie** [Kupon-Protocol/kupon-protocol-contracts#3]
The `claim()` method in KuponNft enables an NFT holder to claim/request an offer that the NFT offers. When a holder triggers the `claim()` function, the NFT is burned and the NFT issuer sees that claim in their Admin dashboard in the front-end.

When the issuer provides the offer to the (former) holder, the issuer can then mark the claim as complete.

I'm wondering whether `claim()` is the best name for that function... How about renaming it to `requestOffer()`?

@jaredalonzo any thoughts on this?

**Comment by jaredalonzo** [comment]
Makes sense. Although the burning of the token doesn't sound quite right with request imo. Request sounds more of a passive stance than asserting something (which they should if they have burned the token). Maybe something like "submit"?

**Comment by jaredalonzo** [comment]
Maybe something like submit()?

**Comment by tempe-techie** [comment]
Maybe `claimOffer()` then? Because claim is a bit more assertive than request.

Submit may not be descriptive enough, I'm afraid...

**Comment by jaredalonzo** [comment]
Yessir! Sounds good to me :)
